src/sentry/static/sentry/app/components/platformList.jsx

```diff
@@ -67,7 +67,6 @@ class PlatformList extends React.Component {
 const getOverlapWidth = size => Math.round(size / 4);
 
 const PlatformIcons = styled('div')`
-  margin-right: ${space(0.5)};
```

Comment:
this was already passed in from the child, but wasn't getting there because we weren't moving `className` through